grype/presenter/table/presenter.go

```diff
@@ -34,7 +34,7 @@ func NewPresenter(results match.Matches, packages []pkg.Package, metadataProvide
 func (pres *Presenter) Present(output io.Writer) error {
 	rows := make([][]string, 0)
 
-	columns := []string{"Name", "Installed", "Fixed-In", "Vulnerability", "Severity"}
+	columns := []string{"Name", "Type", "Installed", "Fixed-In", "Vulnerability", "Severity"}
```

Comment:
open question @sparrowt @luhring @spiffcs : should we try to be consistent with syft, which is `name, version, type ...` thus change this to `name, installed, fixed-in, type ...`,... or not?

Reply:
Yea I think consistency here between the tools is good and then we are good to merge this PR.

Reply:
Okey doke, moved in 14177cc